### ISSUE TITLE
error for PCRdupestimate

### ISSUE THREAD

**Issue by MoustafaAbohawya** [vibansal/PCRduplicates#6]
Hello, 
I am using the program to estimate the PCR duplicates in exome seq data. However, the first code run smoothly. The second is not running I would say due to problem with python interpreter. I am using Python 3.9.13. When I run

estimate_PCRduprate.py -i sample.hetreads -f exome > sample.PCRdups

It gives me an error. 

for c in xrange(MAX_CLUSTER_SIZE): duplicate_counts.append([0,0.0,0.0,0.0]);
TabError: inconsistent use of tabs and spaces in indentation

Is the software compatible only certain version of python? Any suggestion?

**Comment by vibansal** [comment]
Yes, it was written some time ago and the python code was tested with Python 2.7. I don't think that it will work with Python 3.9 without modifications.
